OCIF/staking/src/lib.rs

```diff
@@ -504,9 +530,12 @@ pub mod pallet {
                 <T as Config>::MaxStakersPerCore::get().div(100) * <T as Config>::WeightInfo::unstake()
         )]
         pub fn unregister_core(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
+            // whats left to do is copy the storage to a temp storage for the unregistering
+            // set the stake info to zero.
+            // create the call to unregister parts, integrate pallet_messages.
```

Comment:
Is this not done?

Reply:
it is, just forgot to delete the comments I wrote when I was structuring stuff